### ISSUE TITLE
Improve website usability

### ISSUE THREAD

**Issue by twler** [json-schema-form/angular-schema-form#811]
#### Enhancement
It's hard to navigate the docs site, particularly the homepage schemaform.io and the examples pages.  This adds to frustration each time and decreases likelyhood that people will use the tool.

#### Expected behaviour
I expected an easy-to-find docs button on the home page, and reasonably sized font on examples page (it is huge).

#### Actual behaviour
It requires scrolling on home page to even find the docs or demo button.  Screen space at the top is instead dedicated to a carousel that mocks itself for being a useless picture of wheat.  Mildly amusing the first time I saw it; but being honest it makes me want to find an alternative every time since.

Examples page is borderline unusable because the code samples look huge.

#### Gist/Plunker/Demo
Not useful (at least add demos / docs buttons to the top):

<img width="1264" alt="screen shot 2016-12-05 at 10 14 35 am" src="https://cloud.githubusercontent.com/assets/819876/20896666/ab412086-bad3-11e6-83c9-090c9dd2ea71.png">

Screen view on macbook pro screen.  Code font is far too large for panel size.  Scrolling up and down is a pain when trying to read the schema:

<img width="1266" alt="screen shot 2016-12-05 at 10 12 08 am" src="https://cloud.githubusercontent.com/assets/819876/20896572/5168f3f4-bad3-11e6-9e05-4a067729af2a.png">

@json-schema-form/angular-schema-form-lead

**Comment by RPGillespie6** [comment]
This is very true. It's extremely difficult to find a complete working example anywhere. Even the demo page just gives you JSON, when what I really need is the code that goes into the angular module, controller, html, etc. The JSON is the easy part.

A complete, minimal JSFiddle would be a good start.

**Comment by Anthropic** [comment]
@tyrw agreed, thanks for the feedback, I've had the same thoughts myself. I'm looking into dgeni to see if we can generate a documentation based site at some point, but we rely on community contributions, so unless someone is willing to mock up a basic website for us or make a PR I have to focus on functionality for work more than anything else.

@RPGillespie6 check this Plunker out https://plnkr.co/edit/hukXFi?p=preview, do you think it meets the need you described? @nicklasb just put it together and we will add a link to it from the readme, open to enhancement suggestions too.
ps. it is for the alpha version.

**Comment by tyrw** [comment]
@Anthropic very much understand.  It is noteworthy though that in the time since I made the initial comment, we ended up rolling our own literally because the documentation is too painful to navigate.  The main culprit honestly is the giant font size for the schema page.  It seems like that would be a 5 minute fix, but I don't have a dog in the fight anymore.  Thanks for running the project either way!

**Comment by nicklasb** [comment]
@RPGillespie6 
It should be noted, WRT dependencies, that they are listed on the front page of the git repo:
https://github.com/json-schema-form/angular-schema-form#script-loading

So they also include angular and angular-sanitize. 
And that ObjectPath is included in >1.0.

**Comment by RPGillespie6** [comment]
@Anthropic Looks good to me. That would have saved me a lot of time having a complete working example and I'm sure it will help others.

@nicklasb That's where I eventually found them, however the landing page for this library from google for me was http://schemaform.io/ and from that landing page I spent a lot of time trying to get an example working from the demo and documentation. If I had known the github readme was more useful than the demo and documentation for bootstrapping yourself I would have gone there first, but alas.

**Comment by nicklasb** [comment]
@RPGillespie6  Ok, I guess I am a bit github-centric, perhaps. But of course there should be a link.

**Comment by scottux** [comment]
@Anthropic I am familiar with dgeni and can make some minor tweaks to the examples page some time this week.
